central/notifiers/acscsemail/acscsemail.go

```diff
@@ -137,17 +133,8 @@ func init() {
 		return
 	}
 
-	cryptoKey := ""
-	var err error
-	if env.EncNotifierCreds.BooleanSetting() {
-		cryptoKey, _, err = notifierUtils.GetActiveNotifierEncryptionKey()
-		if err != nil {
-			utils.Should(errors.Wrap(err, "Error reading encryption key, notifier will be unable to send notifications"))
-		}
-	}
-
 	notifiers.Add(notifiers.ACSCSEmailType, func(notifier *storage.Notifier) (notifiers.Notifier, error) {
-		g, err := newACSCSEmail(notifier, ClientSingleton(), metadatagetter.Singleton(), mitreDS.Singleton(), cryptocodec.Singleton(), cryptoKey)
```

Comment:
@charmik-redhat @johannes94 Can this code be removed from ACSCS E-Mail notifier? The variables were never used in the notifier directly.
Is this something we need to fix or can safely be removed? AFAIK currently the ACSCS E-Mail credential is not encrypted (questionable if it needs them if SA auth is used).

Reply:
Yes, the ACSCS E-Mail sender does only use the SA no credentials stored in the database.

I was unsure about whether calling code would require this to be set or not when I implemented this feature.

It looks like the compatibility GH action for emailsender compatibility is just working fine, so it should be ok to remove this.